lib/datapacksjob.js

```diff
@@ -538,7 +538,32 @@ DataPacksJob.prototype.formatResponse = async function(jobInfo, error) {
                         VlocityUtils.verbose(dataPack.VlocityDataPackKey, dataPack.VlocityDataPackStatus); 
                     }
                 }
-
+                if(Object.keys(allManifestByKey).length !== 0){
```

Comment:
This chunk should be in `formatResponse` where it can be a part of the other DataPack data getting injected in the large `else` statement. It should be as straightforward as if there is an "Id" then add the orgUrl. This spot is similar but then you are iterating over the manifest vs sourceData, but this should not be necessary as it already happens in formateResponse.

Reply:
Iterations have been removed from formatResponse function.